database.md

```diff
@@ -285,6 +285,8 @@ If you would like to specify a closure that is invoked for each SQL query execut
         }
     }
 
+Inside `DB::listen` you should not perform any `INSERT` operations otherwise the Eloquent `save` method will not work correctly when saving new models in the database.
```

Comment:
```suggestion
> **Warning**  
> Inside `DB::listen` you should not perform any `INSERT` operations otherwise the Eloquent `save` method will not work correctly when saving new models in the database.
```
![изображение](https://github.com/laravel/docs/assets/13885653/901b0f32-8044-4437-b9d0-5a32fc13df11)